### ISSUE TITLE
Feature request: the City-state "Manila" be renamed to "Maynila"

### ISSUE THREAD

**Issue by Issues66010222** [yairm210/Unciv#11436]
### Before creating

- [X] This is NOT a gameplay feature from Civ VI, BNW, or outside - see Roadmap
- [X] This is NOT a gameplay feature from Vanilla Civ V or from G&K - If so, it should be a comment in https://github.com/yairm210/Unciv/issues/4697

### Problem Description

Since the CS `Manila` is described as a `"Maritime"` cityStateType, that must mean it's referring to the pre-colonial polity known as **Maynila**; "Manila", as we know it as today, only existed since the pre-modern times: as a Spanish colonial city, and later on as the modern capital city of The Philippines.

There is/are mod(s) out there that adds Manila as a city or capital city for a Nation.

### Desired Solution

Basically, this is a spelling correction: change "Manila" to "Maynila". Also change the adjective form "Manilan" - which seems invalid in my opinion - to "Tagalog" (plural: Tagalogs).

The reason I put this up as a Suggestion rather than a Bug Fix is because this is more-or-less inconsequential, and there are already way too many Bug Reports that I feel like this should take less priority.

### Alternative Approaches

(I don't know what to put here as "alternative approaches")

**Comment by yairm210** [comment]
Changing a civ name will break existing games, since that's also the *id* of that civ - so cities and units belonging to Manila will not find who they belong to
There's no problem changing city names etc, since those are purely cosmetic - and there's also no problem if a civ has a city in the same name as another civ, as you're describing here.
TL;DR I see your point, but the detrimental effects would far outweigh the slight historical inaccuracy we have here, so I'm politely declining this change :)

What's more is that this *is* the [Civ V](https://civilization.fandom.com/wiki/Manila_(Civ5)) name for it. 

You are of course welcome to change this in mods
